content/blog/2018/06/2018-06-19-new-design-digest.adoc

```diff
@@ -0,0 +1,162 @@
+---
+layout: post
+title: "New design and UX digest"
+tags:
+- core
+- developer
+author: scherler
+---
+
+== Overview
+
+The recent changes to some core pages provide new design and UX and further dropping all external dependencies to prevent
+any possible malicious javascript introduced by third party libraries. https://hackernoon.com/im-harvesting-credit-card-numbers-and-passwords-from-your-site-here-s-how-9a8cb347c5b5[Why?]
+
+This meant to drop the usage of the jelly layout lib (aka xmlns:l="/lib/layout") and as well the page decorators it
+supported. However there is a new SimplePageDecorator which can be used to modify the look and feel for the login and sign up page.
+
+The following pages have given a new design:
+
+- HudsonIsLoading / HudsonIsRestarting
+
+image:/images/post-images/2018-06-19-JENKINS-50447/JENKINS-50447_1_a.png[role="center"]
+
+image:/images/post-images/2018-06-19-JENKINS-50447/JENKINS-50447_1_b.png[role="center"]
+
+- Login
+
+image:/images/post-images/2018-06-19-JENKINS-50447/JENKINS-50447_2.png[role="center"]
+
+- Sign up
+
+image:/images/post-images/2018-06-19-JENKINS-50447/JENKINS-50447_3.png[role="center"]
+
+
+== UX enhancement
+
+Form validation has changed to give inline feedback about data validation errors in the same form.
+
+- Login
+
+image:/images/post-images/2018-06-19-JENKINS-50447/JENKINS-50447_4.png[role="center"]
+
+- Sign up
+
+image:/images/post-images/2018-06-19-JENKINS-50447/JENKINS-50447_5.png[role="center"]
+
+The above image shows that the validation is now done on all input fields instead of before breaking on the
+first error found, which should lead to less retry cycles.
+
+Instead of having the user repeat the chosen password the new ux introduces the possibility to see the password
+and further implemented a basic password strength meter
```

Comment:
I do not follow this sentence, is there a word missing or something?
At least, we should add a comma to make it more readable.

Something along 

> Instead of having the user repeat the chosen password, the new ux introduces the possibility to see the password [...]

IIUC, I think you should add a period, and add a dedicated sentence about the strength meter.